### DISCUSSION TITLE
[question]: does the web bundle include imports like react-navigation or safe-area?

### DISCUSSION THREAD

**Discussion by cloudorbush** [nandorojo/solito#42] [Q&A] [answered]
Hi @nandorojo ,

Thanks for building an awesome piece of tech.

I see that the `package.json` file in next app imports `"app": "*",` as one of its dependencies.

Does that mean that libraries like `react-navigation` or `react-native-safe-area`, which are under the dependencies in `packages/app` but are a no-op on web, will still be included in the next app bundle?

Should I be worried about bloating the web bundle size like this by adding packages to `package.json` under `packages/app`?

**Comment by nandorojo** [comment, marked as answer]
good question. I don’t think so. I’ve run tests with next’s webpack bundle analyzer and react navigation doesn’t show as getting imported anywhere / affecting the page sizes. i recommend confirming this with the next bundle analyzer by importing / not importing the safe area library and comparing the difference. i made sure to tree shake solito for each platform as much as i could. it’s just up to you to make sure that you don’t import for the wrong platform (notice the .web files in some places to achieve this)

hope that helps!
